client/src/components/MultiJurisdictionAudit/Progress/JurisdictionDetail.tsx

```diff
@@ -138,24 +139,43 @@ const RoundStatusSection = ({
   const [auditBoards] = useAuditBoards(electionId, jurisdiction.id, [round])
   const ballots = useBallots(electionId, jurisdiction.id, round.id, auditBoards)
   if (!auditBoards || !ballots) return null
+
+  const status = (() => {
+    const jurisdictionStatus =
+      jurisdiction.currentRoundStatus && jurisdiction.currentRoundStatus.status
+
+    if (round.sampledAllBallots) {
+      if (jurisdictionStatus === JurisdictionRoundStatus.COMPLETE)
+        return <p>Data entry complete</p>
+      if (auditBoards.length === 0)
+        return <p>Waiting for jurisdiction to set up audit boards</p>
+      return (
+        <p>Auditing all {jurisdiction.ballotManifest.numBallots} ballots</p>
```

Comment:
I'm seeing `jurisdiction.ballotManifest.numBallots` here, and `jurisdiction.numBallots` elsewhere. Is that right? Are these kept in sync?

Reply:
- `jurisdiction.numBallots` is part of the JA /api/me response
- `jurisdiction.ballotManifest.numBallots` is part of the AA /jurisdiction response

So different endpoints for different user types. They come from the same db field, yes.

Reply:
If you think it makes more sense to return the whole `ballotManifest` object to the JA as well, I'm open to that. Adding just `numBallots` was the quickest/smallest change, but I didn't think it through deeply